javascript/src/main/pages/Home/Home.js

```diff
@@ -1,31 +1,24 @@
 import React from "react";
 import { Jumbotron } from "react-bootstrap";
+import {useAuth0} from "@auth0/auth0-react";
+import Redirect from "react-router-dom/es/Redirect";
```

Comment:
I think this should be imported as follows:
```suggestion
import { Redirect } from "react-router-dom";
```

Reply:
Yep, that's correct! Fixed